README.md

```diff
@@ -111,7 +111,8 @@ In your `macros` directory, create a file called `ref.sql` with the following co
     prod_schema=var("upstream_prod_schema", None),
     enabled=var("upstream_prod_enabled", True),
     fallback=var("upstream_prod_fallback", False),
-    env_schemas=var("upstream_prod_env_schemas", False)
+    env_schemas=var("upstream_prod_env_schemas", False),
+    version=None
 ) %}
 
   {% do return(upstream_prod.ref(parent_model, prod_database, prod_schema, enabled, fallback, env_schemas)) %}
```

Comment:
Should we pass the version as an argument in this function call?

```suggestion
  {% do return(upstream_prod.ref(parent_model, prod_database, prod_schema, enabled, fallback, env_schemas, version)) %}
```

Reply:
Good catch, thanks!